api/server/v1/marshaler.go

```diff
@@ -532,12 +532,11 @@ func (x *DescribeCollectionResponse) MarshalJSON() ([]byte, error) {
 
 func (x *DescribeDatabaseResponse) MarshalJSON() ([]byte, error) {
 	resp := struct {
-		Db          string            `json:"db"`
-		Metadata    *DatabaseMetadata `json:"metadata"`
-		Collections []*collDesc       `json:"collections"`
-		Size        int64             `json:"size"`
+		Project     string           `json:"project"`
+		Metadata    *ProjectMetadata `json:"metadata"`
```

Comment:
This shouldn't contain the ProjectMetadata because Project is a higher level construct that would have more things than a database.

Reply:
approved PR 168

Reply:
thanks - now https://github.com/tigrisdata/tigris-client-go/pull/193

Reply:
updated the current PR.